guides/common/modules/ref_system-requirements.adoc

```diff
@@ -40,6 +40,11 @@ endif::[]
 * Administrative user (root) access
 * Full forward and reverse DNS resolution using a fully-qualified domain name
 
+{Project} only supports `UTF-8` encoding system.
+No other encoding method is supported.
+Set `en_US.utf-8` in System-wide locale settings if your language is English and territory is USA.
```

Comment:
```suggestion
For example, set `en_US.utf-8` as the system-wide locale if your language is English and territory is USA.
```

Reply:
@ekohl Does the user have to specify the territory or can they use just `utf-8`?

Reply:
We'd generally start with "If X, then Y", so "For example, if ..., set ...".

Reply:
@Lennonka a locale is always a combination. https://en.wikipedia.org/wiki/Locale_(computer_software)#POSIX_platforms says it's defined as `[language[_territory][.codeset][@modifier]]`.

A bit of backstory for those who don't know the potential impact of this.

Many systems use `en_US.UTF-8` and a lot of software doesn't even consider other locales may behave differently. For example, it may affect sorting:
```console
$ echo -e "a\nb\nc\nch\nd\nh\ni" | LC_ALL=en_US.UTF-8 sort
a
b
c
ch
d
h
i
$ echo -e "a\nb\nc\nch\nd\nh\ni" | LC_ALL=cs_CZ.UTF-8 sort
a
b
c
d
h
ch
i
```

Similarly, it affects how dates are printed (try `LC_TIME=... date` or `LC_ALL=...`). https://github.com/theforeman/foreman-installer/commit/926628891b02f7408cd59ce7c0a832c94709f3b4 is an example of a bug that was solved. It tried to parse a specifically formatted date, which worked in one locale but failed in another. There the easiest solution was to not store & parse the date, but rather format it just once.

The backstory of this PR is that some dependency of Pulpcore (I can't remember which one right now) would segfault if the ISO-8859-1 encoding was used instead of UTF-8.

Locales, like timezones, are a difficult subject filled with bugs because it's always more complex than you think.

Reply:
> 

Addressed :+1: